### PR TITLE
fix(text): set all default styles

### PR DESCRIPTION
**Pull request by yinonov**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/text/src/vwc-text.scss [hunk 1]
```diff
@@ -1,49 +1,80 @@
...
+	}
+
+	@return '&:not(#{$classes})';
+}
+
+$default-selector: get-default-selector($font-faces);
+
+%not-tight {
+	&:not(.tight) {
+		display: block;
```
Comment (yinonov) [inline components/text/src/vwc-text.scss:30]: @rachelbt what's the criterion for display block of text?

File: components/text/src/vwc-text.scss [hunk 2]
```diff
@@ -1,49 +1,80 @@
...
-			display: block;
-			margin: var(--font-face-margin, 0);
+	#{$default-selector},
+	&.font-face-body-1,
+	&.font-face-body-1-bold,
+	&.font-face-body-1-code,
+	&.font-face-body-1-link,
+	&.font-face-body-2,
+	&.font-face-body-2-bold,
+	&.font-face-body-2-code,
```
Comment (rachelbt) [inline components/text/src/vwc-text.scss:70]: code & link cant get block. they need inline-block

File: components/text/src/vwc-text.scss (hunk 1 above)
Comment (rachelbt) [reply components/text/src/vwc-text.scss:30]: as in the user agent stylesheet

**Review by rachelbt — COMMENTED** [review]
.

**Pull request closed without merging**